baselines/pFedHN/pFedHN/main.py

```diff
@@ -0,0 +1,88 @@
+"""Main script for pFedHN."""
+
+import flwr as fl
+import hydra
+import torch
+from flwr.server.client_manager import SimpleClientManager
+from hydra.utils import instantiate
+from omegaconf import DictConfig
+
+from pFedHN.client import generate_client_fn
+from pFedHN.dataset import gen_random_loaders
+from pFedHN.models import LocalLayer
+from pFedHN.server import pFedHNServer
+from pFedHN.utils import get_device, set_seed
+
+
+@hydra.main(config_path="conf", config_name="base", version_base=None)
+def main(cfg: DictConfig):
+    """Run the baseline.
+
+    Parameters
+    ----------
+    cfg : DictConfig
+        An omegaconf object that stores the hydra config.
+    """
+    set_seed(42)
+
+    # partition dataset and get dataloaders
+    # pylint: disable=unbalanced-tuple-unpacking
+    trainloaders, valloaders, testloaders = gen_random_loaders(
+        cfg.dataset.data,
+        cfg.dataset.root,
+        cfg.client.num_nodes,
+        cfg.client.batch_size,
+        cfg.client.num_classes_per_node,
+    )
+
+    if cfg.model.variant == 1:
+        node_local_layers = [
+            LocalLayer(n_input=84, n_output=cfg.model.out_dim).to(get_device())
```

Comment:
```suggestion
            LocalLayer(n_input=84, n_output=cfg.model.out_dim)
```

Reply:
Yes , I removed it here and in the client side , I moved it to the device